crates/languages/src/go.rs

```diff
@@ -485,8 +485,9 @@ impl ContextProvider for GoContextProvider {
 
         let snapshot = location.buffer.read(cx).snapshot();
         let point_range = location.range.to_point(&snapshot);
+        let start = Point::new(point_range.start.row, 0);
         let end = Point::new(point_range.start.row + 1, 0);
-        let line = snapshot.text_for_range(point_range.start..end).peek();
+        let line = snapshot.text_for_range(start..end).peek();
 
         let go_subtest_variable = extract_subtest_name(line.unwrap_or(""))
```

Comment:
I think `peek()` does not return the whole line. I debugged `line` and it has not contain all line content. I tried to use `collect` and then got the whole line. Could you take a look for this issue?
```
let line = snapshot.text_for_range(start..end).collect::<String>();

let go_subtest_variable = extract_subtest_name(line.as_str())
```

Reply:
It does for me. 

```
[crates/languages/src/go.rs:492:9] &line = Some(
    "\tt.Run(\"Foobar\", func(t *testing.T) {\n",
)
[crates/languages/src/go.rs:492:9] &line = Some(
    "\t\tfmt.Println(\"subtest\")\n",
)
```